releases/release-1.22/release-notes/maps/pr-102673-map.yaml

```diff
@@ -0,0 +1,2 @@
+pr: 102673
+releasenote: {}
```

Comment:
Mhh interesting case, I think this is a bug, since the map does not have any effect perhaps it should not be written at all.

Reply:
@puerco Any suggestion on what to do with this map? Set to the not publish or just delete the map?

Reply:
We can just delete it for cleanliness but it has absolutely no effect. Could you file an issue in kubernetes/release so that I can take a look later?